src/proxy_handler.cc

```diff
@@ -128,7 +202,36 @@ ProxyHandler::HandleRequest(const Request& request, Response* response)
       header.pop_back();
 	  std::string head = header.substr(0, header.find(":"));
 	  std::string value = header.substr(header.find(":") + 2);
-	  response->AddHeader(head, value);
+
+	  if(head == "Cache-Control" 
+	     && value.find("no-store") == std::string::npos 
```

Comment:
That would good to implement. I planned to do that originally but this basic proxy cache took me longer than I thought it would. How difficult / time consuming do you think it would be to implement Etag?